examples/react-websocket/package.json

```diff
@@ -0,0 +1,62 @@
+{
+  "name": "react-websocket",
+  "private": true,
+  "version": "0.0.0",
+  "type": "module",
+  "scripts": {
+    "dev": "concurrently -r 'pnpm dev:server' 'pnpm basic-websocket' 'pnpm y-redis'",
+    "start": "concurrently -r 'pnpm start:server' 'pnpm basic-websocket' 'pnpm y-redis'",
+    "dev:server": "nodemon -w src/server -x tsx src/server/main.ts",
+    "start:server": "NODE_ENV=production pnpm tsx src/server/main.ts",
+    "basic-websocket": "node --env-file .env.basic node_modules/y-websocket/bin/server.cjs",
+    "y-redis": "concurrently -r 'pnpm y-redis-server' 'pnpm y-redis-worker' 'pnpm redis' 'pnpm minio'",
+    "y-redis-server": "wait-on tcp:6379 && node --env-file .env.y-redis node_modules/@y/redis/bin/server.js",
+    "y-redis-worker": "wait-on tcp:6379 && node --env-file .env.y-redis node_modules/@y/redis/bin/worker.js",
+    "redis": "docker run --rm -p 6379:6379 redis",
+    "minio": "docker run --rm -p 9000:9000 -p 9001:9001 quay.io/minio/minio server /data --console-address \":9001\"",
+    "build": "vite build",
+    "lint": "eslint . --ext ts,tsx --report-unused-disable-directives --max-warnings 0",
+    "postinstall": "cp node_modules/@y/redis/.env.template .env.y-redis && sh gen-key.sh"
+  },
+  "dependencies": {
+    "@blocksuite/blocks": "0.14.0-canary-202403201020-e0ac77d",
+    "@blocksuite/presets": "0.14.0-canary-202403201020-e0ac77d",
+    "@blocksuite/store": "0.14.0-canary-202403201020-e0ac77d",
+    "@y/redis": "github:y-js/y-redis",
```

Comment:
This leads to following error when `pnpm i` in the `/examples` path locally, @L-Sun could you reproduce this or any workaround expected?

``` sh
➜  examples git:(example-react-y-redis) pnpm i      
Scope: all 11 workspace projects
Lockfile is up to date, resolution step is skipped
Packages: +1353
++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
 ERROR  Command failed with exit code 128: /usr/bin/git clone git@github.com:y-js/y-redis.git /Users/ewind/Library/pnpm/store/v3/tmp/_tmp_3179_2e2929f352d7f64d714a46cbc5dc4a75
Cloning into '/Users/ewind/Library/pnpm/store/v3/tmp/_tmp_3179_2e2929f352d7f64d714a46cbc5dc4a75'...
git@github.com: Permission denied (publickey).
fatal: Could not read from remote repository.

Please make sure you have the correct access rights
and the repository exists.

pnpm: Command failed with exit code 128: /usr/bin/git clone git@github.com:y-js/y-redis.git /Users/ewind/Library/pnpm/store/v3/tmp/_tmp_3179_2e2929f352d7f64d714a46cbc5dc4a75
Cloning into '/Users/ewind/Library/pnpm/store/v3/tmp/_tmp_3179_2e2929f352d7f64d714a46cbc5dc4a75'...
git@github.com: Permission denied (publickey).
fatal: Could not read from remote repository.

Please make sure you have the correct access rights
and the repository exists.
    at makeError (/Users/ewind/.nvm/versions/node/v20.10.0/lib/node_modules/pnpm/dist/pnpm.cjs:24966:17)
    at handlePromise (/Users/ewind/.nvm/versions/node/v20.10.0/lib/node_modules/pnpm/dist/pnpm.cjs:25537:33)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async gitFetcher (/Users/ewind/.nvm/versions/node/v20.10.0/lib/node_modules/pnpm/dist/pnpm.cjs:110935:11)
    at async fetcher (/Users/ewind/.nvm/versions/node/v20.10.0/lib/node_modules/pnpm/dist/pnpm.cjs:123566:16)
    at async run (/Users/ewind/.nvm/versions/node/v20.10.0/lib/node_modules/pnpm/dist/pnpm.cjs:123032:23)
Progress: resolved 1353, reused 1337, downloaded 13, added 13
```

Reply:
`github:y-js/y-redis` -> `github:yjs/y-redis`.  The reason it works on my computer might be because of the cache.

Reply:
`github:y-js/y-redis` has removed.